xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-ui/src/main/resources/XWiki/Mentions/MentionsMacro.xml

```diff
@@ -374,12 +374,15 @@ require(['deferred!ckeditor', 'xwiki-suggestUsers', 'jquery', 'xwiki-meta'], fun
     <property>
       <code>.xwiki-mention {
   background-color: $services.mentions.mentionsColor;
-  border-radius: 10px;
-  padding: 2px 5px 2px 5px;
+  border-radius: @border-radius-small;
```

Comment:
A you rush 10px is `border-radius-small` ? Feels like a change in radius in the screenshots.

Reply:
I'm not against changing the border ratio, but it feels unrelated to the current fix.
Feel free to open a separate improvement for it though :slightly_smiling_face:

Reply:
Changes reverted in https://github.com/xwiki/xwiki-platform/pull/2638/commits/60023d9bf464aeceb15f2fe4155b8d321ccd791d

Reply:
I just tried creating a ticket, and noticed the recommendation pointing to https://jira.xwiki.org/browse/XWIKI-21502 ...
Adding a note on this ticket :D